README.md

```diff
@@ -64,6 +64,25 @@ Flavors can be built in parallel easily with the `build.parallel` target:
 make FLAVORS="<flavor> <other flavor> <...>" build.parallel
 ```
 
+### Building with a specific version of Ceph
+Some distributions can select a specific version of Ceph to install.
+You just have to pad the required version to the ceph release code name.
+
+The required version will be saved in ``CEPH_POINT_RELEASE`` variable (including the version separator).
+The version separator is usually a dash ('-') or an equal sign ('=').
+``CEPH_POINT_RELEASE`` remains empty if no point release is given.
+
+Note that ``CEPH_VERSION`` variable still feature the ceph code name, **luminous** in this example.
```

Comment:
There are many short and choppy paragraphs in this section that are hard to read. Combine them into a single paragraph so it is clear the information is part of the same concept.

Reply:
I disagree. First part is the why we need that, 2nd is how it works, 3rd is just a notice.
Their are all under the same topic which only address this issue. It is clear this about the same concept.